core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/data/impl/DataFeedServiceImpl.java

```diff
@@ -60,6 +60,8 @@
 public class DataFeedServiceImpl implements DataFeedService {
     private static final Logger LOG = LoggerFactory.getLogger(DataFeedServiceImpl.class);
 
+    private static final CopyOnWriteArrayList<GridResource> GRID_RESOURCE_COPY_ON_WRITE_ARRAY_LIST = new CopyOnWriteArrayList<>();
```

Comment:
(cosmetic) Just `GRID_RESOURCES` ?) What is worth copying the whole loooong name of class into the name of constant if everybody can read the name of class in the same line anyway?

Reply:
+also: 
This does not look like a static field to me. Static field initialized upn the loadin of the class into classloader. It is  destroyed upon the unloading the class from the classloader (therefore. never). So, event stopping the bundle will not clear this "cache". Probably, not a big deal in this particular projecvt, but.. what for?..

This class here is a service. A service is a natual-born singleton (unless created via a config factory, etc. which is not our case). The "usual" fields of a singleton are effectively "singleton-ish" as well. So there is no reason to make them static. It just produces more tricky cases (see above).